ci-plan.md

```diff
@@ -0,0 +1,44 @@
+# Plan for adding embedded CI tests to `rust-lang/rust`
+
+* Goal: we will add target `thumbv7m-none-eabi` in stable Rust 2018 Edition
```

Comment:
We will want to test all four of the Cortex-M targets:

```
thumbv6m-none-eabi - Bare Cortex-M0, M0+, M1
thumbv7em-none-eabi - Bare Cortex-M4, M7
thumbv7em-none-eabihf - Bare Cortex-M4F, M7F, FPU, hardfloat
thumbv7m-none-eabi - Bare Cortex-M3
```

Reply:
Also we will want to test `msp430-none-elf` eventually, but we will focus on the `thumb*` targets first

Reply:
Very important point. Now our target became clearer.

Reply:
For msp430-none-elf, we will need to add ‘dist’ job.

Reply:
In a big picture, the current status is:
- thumb* targets are already in Tier 2 because it compiles and official binaries available.
- msp430-none-elf is Tier 3 because it has no official binary.